.github/workflows/apt-release.yml

```diff
@@ -0,0 +1,61 @@
+name: Apt Release
+on:
+  workflow_dispatch:
+  workflow_run:
+    # wait for Cli Release to complete, when it completes, start this workflow
+    workflows: ["Cli Release"]
+    branches: [main]
+    types:
+      - completed
+
+jobs:
+  apt-build-release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check Out Repo
+      uses: actions/checkout@v2
+    - name: Fetch tag
+      id: get-tag
```

Comment:
Where are we using `TAG_NAME`?

Reply:
yeah I missed this, removed now. 👍 for pointing it out.